packages/aws-cdk/lib/cli/telemetry/sanitation.ts

```diff
@@ -0,0 +1,78 @@
+import { join } from 'path';
+import * as fs from 'fs-extra';
+import { FeatureFlag } from './feature-flags';
+import type { Context } from '../../api/context';
+
+export const CLI_TYPE_REGISTRY_FILE = 'cli-type-registry.json';
+
+/**
+ * argv is the output of yargs
+ */
+export function sanitizeCommandLineArguments(argv: any): { path: string[]; parameters: { [key: string]: string } } {
+  // Get the configuration of the arguments
+  const config = fs.readJSONSync(join(__dirname, '..', CLI_TYPE_REGISTRY_FILE));
```

Comment:
I believe this won't bundle properly. @kaizencc it might be why your integ tests are failing. Consider using `require('cli-type-registry.json')`.